docs/source/contributing/tests.rst

```diff
@@ -3,18 +3,17 @@
 ===================================
 Testing JupyterHub and linting code
 ===================================
-
-Unit tests help confirm that JupyterHub works as intended, including after modifications are made. Additionally, they help in clarifying our expectations for our code.
+Unit test helps to confirm that JupyterHub works as it is supposed to work even when the modifications are made . Additionally they clarify us about our expectations from the code. 
 
 JupyterHub uses `pytest <https://pytest.org>`_ for all the tests. You
-can find them under the `jupyterhub/tests <https://github.com/jupyterhub/jupyterhub/tree/main/jupyterhub/tests>`_ directory in the git repository.
+can find them under ``jupyterhub/tests``  <https://github.com/jupyterhub/jupyterhub/tree/main/jupyterhub/tests>`_ directory in the git repository.
 
 Running the tests
 ==================
+ #Make sure you have completed :ref:`contributing/setup`. Once you are done , you would be able
+ to run ``jupyterhub`` from the command line & access it from your
+ web browser too. This ensures that the dev environment is properly set
```

Comment:
Remove extra `too` on the end. The last line has been lost (`...up for tests to run`), so this sentence doesn't finish.